README.rst

```diff
@@ -19,34 +19,52 @@ from the protobuf source IDL and additional configuration in YAML files.
 Installation
 ------------
 
-At the moment, this library is under development, so please see
-`using a development checkout`_ for installation instructions
+You can now install artman directly from pip:
 
-.. _`using a development checkout`: https://github.com/googleapis/artman/blob/master/CONTRIBUTING.rst#using-a-development-checkout
+.. code::
+
+    pip install googleapis-artman
+
+You may need root privileges if you are not installing inside of a virtualenv.
+This will make the ``artman`` command available on your system.
 
 
 Usage
 -----
 
-Currently, this tool can only be run in the development environment of the
-development team, as some dependencies are yet to be published. See USAGE_ for
-details.
+Before you can use artman, you will need a configuration file. You can run
+``artman init`` to create a simple configuration file.
+
+For building a GAPIC (the most common task), the usage looks like:
+
+.. code::
+
+    artman build --api pubsub --language python
+
+This assumes that you have checkouts of both `googleapis`_ and `toolkit`_
+on your system (and that toolkit is able to run; e.g. you need Java).
```

Comment:
There actually will be more dependency based on language. Maybe refer to Dockerfile as a canonical installation script.